### DISCUSSION TITLE
Github Actions and maintenance branches

### DISCUSSION THREAD

**Discussion by codekult** [semantic-release/semantic-release#3525] [Configuration] [answered]
Hi, I need help to configure Github actions to work with pushes to maintenance branches. Currently we're running semantic-release on every push to `main` branch:

```yaml
on:
  push:
    branches:
      - main
```

Our maintenance branches are defined following the semantic-release default

```js
branches: [
  '+([0-9])?(.{+([0-9]),x}).x',
  'main'
]
```

How can I run semantic-release on every push to a maintenance branch?

**Comment by travi** [comment, marked as answer]
You will need to configure your actions workflow to trigger on branches matching the same patterns. You can find the details of how to do that in the actions docs: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore
